src/v2i-hub/CARMAStreetsPlugin/src/CARMAStreetsPlugin.h

```diff
@@ -155,6 +161,14 @@ class CARMAStreetsPlugin: public PluginClient {
 	 * @brief Intersection Id for intersection
 	 */
 	std::string _intersectionId = "UNSET";
+	/**
+	 * @brief Status label for SRM messages skipped due to errors.
+	 */
+	const char* Key_SRMMessageSkipped = "Failed to convert J2735 SRM To JSON due to errors.";
```

Comment:
https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/74637787cb7971195afe8922725894f2e01feac6/src/v2i-hub/CARMAStreetsPlugin/src/CARMAStreetsPlugin.h#L104 Please fix this to say something like this "SRM message skipped due to errors"